content/create-release.md

```diff
@@ -209,7 +209,7 @@ On a deployed release, a BOSH Agent runs on each job VM.
 BOSH communicates with the Agent, which in turn executes commands in the
 control script.
 The Agent does this using open source process monitoring software called
-[Monit](http://mmonit.com/monit/).
+[Monit, version 5.2.5](https://web.archive.org/web/20110816041503/https://mmonit.co+m/monit/documentation/monit.html).
```

Comment:
Looks like an extraneous `+` character made it in here (`mmonit.com+m`)

Reply:
Indeed, thanks for the eagle-eyed review!